providers/npu/units/jobs.pxu

```diff
@@ -0,0 +1,52 @@
+id: kmd/char_device_permissions
+category_id: npu_kmd
+flags: simple
+_summary: Check that user has read and write access to NPU char device node
+estimated_duration: 2s
+command: check_accel_permissions.py
+imports: from com.canonical.plainbox import manifest
+requires: manifest.has_npu == 'True'
+
+id: kmd/firmware_version
+category_id: npu_kmd
+flags: simple
+imports: from com.canonical.plainbox import manifest
+requires:
+  dmesg_logs_firmware_version.state == 'supported'
+  manifest.has_npu == 'True'
+_summary: Check NPU firmware version
+estimated_duration: 2s
+user: root
+command: check_firmware_version.py
+
+id: umd/check_npu_umd_test
+category_id: npu_umd
+flags: simple
+_summary: Check that npu-umd-test listing runs with no errors
+_description: Verify that the npu-umd-test config file exists and listing using that config file doesn't lead to an error.
+imports: from com.canonical.plainbox import manifest
+requires: manifest.has_npu == 'True'
+command:
+   test -e "$NPU_UMD_TEST_CONFIG" && \
+    intel-npu-driver.npu-umd-test -l --config "$NPU_UMD_TEST_CONFIG"
```

Comment:
Is there a special config file that doesn't require setting up everything [here](https://github.com/canonical/intel-npu-driver-snap?tab=readme-ov-file#running-the-npu-umd-test-application)? The tests do work but I have to manually cd into the snap directory and curl the model XML file